src/store/peer/sagas.ts

```diff
@@ -1,8 +1,20 @@
 import {all, call, put, select, takeLatest} from 'redux-saga/effects';
-import {ApiError, ApiResponse, DeleteResponse, RequestPayload} from '../../services/api-client/types';
+import {
+  ApiError,
+  ApiResponse,
+  ChangeResponse,
+  CreateResponse,
+  DeleteResponse,
+  RequestPayload
+} from '../../services/api-client/types';
 import { Peer } from './types'
 import service from './service';
 import actions from './actions';
+import {Group, GroupPeer} from "../group/types";
```

Comment:
Codacy has a fix for the issue: [Replace `Group,·GroupPeer` with `·Group,·GroupPeer·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import { Group, GroupPeer } from "../group/types";
```